Layers/Renderers/Terrain/Materials/TerrainShader.gdshader

```diff
@@ -223,8 +223,8 @@ void fragment() {
 	float count = 0.0;
 	
 	// Get detail texture sample, but blur the position to get nice transitions
-	for (float x_off = -1.0; x_off <= 1.0; x_off += 0.5) {
-		for (float y_off = -1.0; y_off <= 1.0; y_off += 0.5) {
+	for (float x_off = -1.0; x_off < 2.0; x_off += 0.5) {
```

Comment:
this seems to have been reverted in a merge, please change it back (this results in an asymmetric blur)